tests/test_pyiges.py

```diff
@@ -536,7 +538,38 @@ def test_transformation_vtk(trafo):
     m = trafo.GetMatrix()
     assert m.GetElement(2,3) == pytest.approx(5.67397368511119)
 
+
+def test_conic_arc_parse():
+    # For this file, the conic arc cannot be parsed.
+    # This is either because of a wrong format of the file or a bug in the parsing.
+    # Despite the reason is unclear, we use this here to test the robust discarding mechanism
+    iges = pyiges.read(os.path.join(DIR_TESTS_REFERENCE_DATA, 'example-arcs.iges'))
+    assert not iges.conic_arcs()
```

Comment:
This tests the discarding mechanism, the entity fails with 
```
Warning: Could not initialize entity from parameters with Parameter section ending an line 71. 
Possibly wrong or (yet) unsupported format. Entity will be discarded.
```
on stdout.

Reply:
Note that this tests the discarding mechanism in that the invallid conic arc does not lead to the exception, and in that the other entities (bslines, circular args) *are* parsed and setup nevertheless, and additionally the file has the new form 2 header format